lib/markdown-preview-view.coffee

```diff
@@ -124,6 +124,12 @@ class MarkdownPreviewView extends ScrollView
 
     @disposables.add atom.config.onDidChange 'markdown-preview.breakOnSingleNewline', changeHandler
 
+    @disposables.add atom.config.observe 'markdown-preview.useGitHubStyle', (useGitHubStyle) =>
+      if useGitHubStyle
+        @element.setAttribute('data-use-github-style', '')
```

Comment:
Super minor, but for data attributes, you could do `@element.dataset.useGithubStyle` instead of `setAttribute` with a `data-` prefix.

https://github.com/atom/atom/blob/1602b8cf45c8fb3c355691387fc4423750210f2c/src/pane-element.coffee#L84-L95